src/api/controllers/team/addUsers.js

```diff
@@ -0,0 +1,70 @@
+const { check } = require('express-validator/check');
```

Comment:
```suggestion
const { check } = require('express-validator');
```
C'est deprecated sinon